packages/blade/src/components/Collapsible/CollapsiblePanel.web.tsx

```diff
@@ -119,7 +120,12 @@ const CollapsiblePanel = ({ children }: CollapsiblePanelProps): ReactElement =>
       defaultIsExpanded={defaultIsExpanded}
       onTransitionEnd={onTransitionEnd}
     >
-      {children}
+      <Box
+        // need margin here to prevent jank
```

Comment:
what is the reason for jank?

Reply:
So we need a margin inside the outside wrapper so this is included in height calculations and prevents jank. Earlier I tried following for spacing which didn't work:

- `gap` on the container flex (gap isn't included in height calculation and led to jank noticed in PR1)
- `paddingBottom` / `marginBottom` on the collapse link (again it wouldn't get counted in the height calculations)

So we needed a margin on the collapsible body's inner container, that ultimately worked 😅